great_expectations/data_context/store/expectations_store.py

```diff
@@ -244,17 +244,28 @@ def get_key(
         self, suite: ExpectationSuite
     ) -> GXCloudIdentifier | ExpectationSuiteIdentifier:
         """Given an ExpectationSuite, build the correct key for use in the ExpectationsStore."""
+        return self._get_key(name=suite.name, id=suite.ge_cloud_id)
+
+    def get_key_by_name(
```

Comment:
It's not immediately clear to me the use case of this vs `get_key`. What does it mean if we are in cloud mode and you call each of these methods, getting back one key with an id and one without?

Reply:
thanks for pointing this out - i've updated the signature of `get_key` in order to remove `get_key_by_name` in [d9fa2cb](https://github.com/great-expectations/great_expectations/pull/9139/commits/d9fa2cbbcca96ec608db33cafea99dc3291f8c82).

Reply:
Thanks!